pilot/pkg/networking/deprecated/config.go

```diff
@@ -15,9 +15,11 @@
 package deprecated
 
 import (
+	"encoding/json"
 	"fmt"
 	"sort"
 	"strings"
+	"time"
```

Comment:
can you do all these changes in networking/v1alpha3/config.go ? I created a copy of that code there just for this purpose.

Reply:
Because we want to be able to throw away this folder (Deprecated/). The more code we add, the more we have to keep track of todos/what was done in one but not in other..

Reply:
I'm confused - what is the purpose of deprecated? If it's not going to be worked on why not just delete it? It it not actively used anywhere.

Reply:
That was my point as well, but costin wants to have that folder around.

Reply:
What I want is to still support v1alpha1 for some time. 

I'm ok with moving all v1alpha3-specific code to the new folder, and keeping the alpha1 in deprecated (until it is gone). 

I don't know if splitting them is best use of the time - my proposal was to just keep the methods for alpha1 around for few weeks and delete them after 0.8 is out. 

But I really don't want to waste more time moving things around  - the goal is to have LDSv2 passing integration tests and getting serious e2e testing, we can move the files around 5 times after.

Reply:
Ok, I think we can close this item. I've migrated the changes to both folders.